components/com_joomgallery/views/image/view.raw.php

```diff
@@ -226,53 +226,78 @@ public function display($tpl = null)
     JResponse::setHeader('Content-disposition', $disposition.'; filename='.basename($img));
 
     // Inlude watermark and crop
-    if(($include_watermark || $crop_image) && !$model->isGif($img))
+    // Create tmp file
+    $tmp_folder = JFactory::getApplication()->get('tmp_path');
+    $img_output   = $tmp_folder.'/tmp_'.basename($img);
+    if($crop_image || $include_watermark)
     {
-      $img_resource = null;
-      if($crop_image)
+      if(!JFile::copy($img, $img_output))
       {
-        $croppos  = JRequest::getInt('pos');
-        $offsetx  = JRequest::getInt('x');
-        $offsety  = JRequest::getInt('y');
-        $img_resource = $model->cropImage($img, $cropwidth, $cropheight, $croppos, $offsetx, $offsety);
+        return $this->displayError(JText::_('COM_JOOMGALLERY_COMMON_MSG_IMAGE_NOT_EXIST'));
       }
+    }
+    else
+    {
+      $img_output = $img;
+    }
 
-      if($include_watermark)
+    // crop image
+    if ($crop_image)
+    {
+      $croppos     = JRequest::getInt('pos');
+      $offsetx     = JRequest::getInt('x');   // what are this variables for?
```

Comment:
Do we need this variables anymore?
Is there anywhere in JoomGallery a usage of this variables?
As far as I can see, this variables are always set to 0 (zero). In this case, these variables can be omitted.